### DISCUSSION TITLE
DRIVING LICENCE NFC

### DISCUSSION THREAD

**Discussion by mohamed00736** [martinpaljak/GlobalPlatformPro#366] [General]
i have a driving licence that is made by global platfor pro i wanna read its info using android device , i used jmrtd and scuba but it doesnt work  , any one can help me please below some technical specs about the card : 
![Screenshot_20241022-103138_TagInfo](https://github.com/user-attachments/assets/592e108e-ab1f-453f-a81b-67b00e107464)

![Screenshot_20241022-103120_TagInfo](https://github.com/user-attachments/assets/5dea8c35-1b51-4682-a277-0a28d2452860)

![Screenshot_20241022-103133_TagInfo](https://github.com/user-attachments/assets/46b4034c-9482-4824-b69f-b910439b3e56)

**Comment by martinpaljak** [comment]
Global platform pro does not make any cards, definitely not drivers licenses. To use GPPro with such a card (for more than gp -i -d -v) the usual main requirement applies: knowledge of security domain keys.
